hparams.py

```diff
@@ -125,6 +125,14 @@
     # Forced garbage collection probability
     # Use only when MemoryError continues in Windows (Disabled by default)
     #gc_probability = 0.001,
+	
+	# json_meta mode only
+	# 0: "use all",
+    # 1: "ignore only unmatched_alignment",
+    # 2: "fully ignore recognition",
+    ignore_recognition_level = 2,
+    min_text=20,
```

Comment:
I was also thinking about this and something like `min_frames` to remove short audio clips from training data. Just out of curiosity, did you get improvements by this? I believe the parameter highly depends on datasets and I'd be happy if you could leave a comment for exmaple: `min_text=20 works good for dataset A but can be adjusted depends on dataset`

Reply:
Actually it was implemented for 2 reasons. 

1. My automatic alignment tool(which I am going to release it soon) cannot handle short speeches well. 
2. From my experience, short speeches in non-dedicated dataset(especially that are extracted from movie clips) were prone to noises, and different cadence of speech. 
(e.g. word "help" in  "The help that is needed is not there." vs. "help" in "HELP!!!")
3. (From my experience with other deep learning based tts) Even if the dataset is nearly noise-free and has uniform cadence, short speeches tend to interfere the result. (probably because my test set is usually at least 3 words long)

But it was implemented as a quick-fix, and I do know that min_frame is much much better solution.

Will leave the comments :)